pages/about.js

```diff
@@ -7,14 +7,253 @@ import { prefixLink } from 'gatsby-helpers';
 export default class About extends Component {
   render () {
     return (
-      <div className="page page--about">
-        <Helmet
-          title={config.siteTitle}
-        />
-        <h1>
-          About
-        </h1>
-      </div>
+	      <div className="page page--about">
+	        <Helmet
+	          title={config.siteTitle}
+	        />
+	 		<div className="page__title">
+	          About
+	     </div>
+	     <div className="page__content">
+	     	<div className="col col--1">History</div>
```

Comment:
Please take a look at the ```_base.scss``` styles again, and how the other pages used the styles in the HTML.  For example, ```col--2``` should not be nested under ```col--1```, but should be a sibling (same indentation)